apps/trading/pages/_app.page.tsx

```diff
@@ -31,31 +32,28 @@ const Title = () => {
     if (networkName) return `${pageTitle} [${networkName}]`;
     return pageTitle;
   }, [pageTitle, networkName]);
-  return (
-    <Head>
-      <title>{title}</title>
-    </Head>
-  );
+  return <title>{title}</title>;
```

Comment:
Title managed like that won't work. It has to be **[direct](https://nextjs.org/docs/api-reference/next/head)** children of <Head /> element